config/locales/recaptcha.en.yml

```diff
@@ -0,0 +1,7 @@
+en:
+  recaptcha:
+    error: ReCaptcha verification failed. Please try again.
+    privacy_message:
```

Comment:
nitpick: if I'm not mistaken, you can add `_html` to the name and won't need to add html_safe on place when you call this locale, like we do in Hub. or do you think that htm_safe is better?

Reply:
I didn't know that. Thank you.